Source/src/UI/Window.cpp

```diff
@@ -1,6 +1,6 @@
-#include "Window.h"
+#include "core/hepch.h"
 
-Hachiko::Window::Window(const char* name, bool active) :
+Hachiko::Window::Window(const char* name, const bool active) :
```

Comment:
What's the point of having const in bool? Is it to further indicate to the maintainer of this method to just use the flag as readonly?

Reply:
this is a futile for this data type, but it is a format that was applied project-wise